### PR TITLE
Auth0 End To End Cypress Tests + screenshot / video generation for failed tests

### PR DESCRIPTION
**Pull request by nickcastel50**
This PR has been updated for the following:

Auth0 End To End Cypress Tests
Screenshot / Video artifact generation for failed tests
One Save IFC test.

[Profile: 100 Login Test](https://github.com/bldrs-ai/Share/issues/1052)

### CODE REVIEW

File: cypress/e2e/saveModelControl/save-model.cy.js [hunk 1]
```diff
@@ -155,7 +163,7 @@ describe('save model', () => {
         cy.wait('@authorizeRequest').its('response.statusCode').should('eq', STATUS_OK)
         cy.wait('@tokenRequest').its('response.statusCode').should('eq', STATUS_OK)
         // Commented out for now
-        // cy.findByTestId('Save IFC', {timeout: 60000}).should('exist')
+        cy.findByTestId('Save IFC', {timeout: 60000}).should('exist')
```
Comment (pablo-mayrgundter) [inline cypress/e2e/saveModelControl/save-model.cy.js]: I feel like these should be short, e.g. 5s.  If we don't have the popup almost immediately, it's bc it's not going to work.  Or that's how it's been for me with the dialogs i was testing recently

if so, the length for these timeouts should be smth like "to make tests that are going to fail, fail faster, but no so short that we get flaky behavior"

File: src/store/AppSlice.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
+const isOpfsEnabled = (process.env.OPFS_IS_ENABLED || 'false').toLowerCase() === 'true'
```
Comment (pablo-mayrgundter) [inline src/store/AppSlice.js]: I think you need to merge?  There's a new condition here based on our convo yesterday https://github.com/bldrs-ai/Share/blob/main/src/store/AppSlice.js

File: .gitignore [hunk 3]
```diff
@@ -15,3 +15,7 @@ cypress/downloads
 *.log
 *~
 \#*
+
+#do not check in cypress videos or screenshots
+cypress/videos/*
+cypress/screenshots/*
```
Comment (pablo-mayrgundter) [inline .gitignore]: per convo, pls remove

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (nickcastel50) [reply .gitignore]: PTAL

File: src/store/AppSlice.js (hunk 2 above)
Comment (nickcastel50) [reply src/store/AppSlice.js]: PTAL

File: cypress/e2e/saveModelControl/save-model.cy.js (hunk 1 above)
Comment (nickcastel50) [reply cypress/e2e/saveModelControl/save-model.cy.js]: PTAL

**Pull request closed without merging**